applications/portal/frontend/src/components/viewers/detailsViewer/DetailsViewer.tsx

```diff
@@ -0,0 +1,483 @@
+                        >
+                            <IconButton>
+                                <img src={INFO_ICON} alt="" />
+                            </IconButton>
+                        </Tooltip>
+                        <Button variant="contained" className={classes.deleteBtn} onClick={() => setOpenDeleteDialog(true)}>Delete file</Button>
+                    </Box>
+                </Box>
+                <Box display="flex" alignItems="center" justifyContent="center" py={4}>
+                    <img src={pageImg} alt='' style={{ maxWidth: 500, maxHeight: 400 }} />
+                </Box>
+            </div>
+            <Box className={classes.pagination}>
+                <Typography>
+                    Page {page} of {data[tabIdx].files[selectedIndex].pages.length}
+                </Typography>
+                <Pagination
+                    count={data[tabIdx].files[selectedIndex].pages.length}
+                    size="large"
+                    page={page}
+                    variant="outlined"
+                    shape="rounded"
+                    onChange={handlePageChange}
+                />
+            </Box>
+            <Modal
+                open={openUploadDialog}
+                dialogActions={true}
+                actionText="Upload"
+                disableGutter={true}
+                handleClose={() => setOpenUploadDialog(false)}
+                handleAction={() => { }}
+                title={`Upload PDF to Population ${populationName}`}
+            >
+                <Box display="flex" alignItems="center" justifyContent="center" p={2}>
+                    <PdfFileDrop
+                        file={files[0]}
+                        setFile={(value: any) => handleAddFile(PDF_FILE, value)}
+                        hasErrors={validationErrors.has(PDF_FILE)}
+                    />
+                </Box>
+                <CategorySelect category={category} handleCategoryChange={handleCategoryChange} />
+                {
+                    files[0] !== undefined && <Box className={classes.addAnotherFileBox}>
+                        <AddPdfFileDrop
+                            file={files[1]}
+                            setFile={(value: any) => handleAddFile(PDF_FILE, value)}
+                            hasErrors={validationErrors.has(PDF_FILE)}
+                        />
+                    </Box>
+                }
+            </Modal>
+            <DeleteDialog
+                open={openDeleteDialog}
+                handleClose={() => setOpenDeleteDialog(false)}
+                actionText="Delete"
+                title="Delete this file?"
+                dialogActions={true}
+                handleAction={() => { }}
+            >
+                <Box className={classes.deleteModalBox}>
+                    <Typography>This action cannot be undone. Are you sure you want to delete “Electrophysiological properties of V2a
```

Comment:
Here again, try to avoid using hardcoded placeholder values. What is "“Electrophysiological properties of V2a interneurons in the lumbar spinal cord.pdf"? Is it the filename? How do we know what's the current file we are seeing?